src/App.tsx

```diff
@@ -186,11 +188,20 @@ export default function App() {
 			<table>
 				<tr>
 					<td><span className={"timeline-cell label"}>Frame</span></td>
-					<td><input className={"timeline-cell"} type={"text"} value={"0"} /></td>
+					<td><input className={"timeline-cell"} type={"number"} value={frameStart} min={"0"} onChange={(e) => 
+							{const inputInt = parseInt(e.target.value);
+							if (inputInt < markerFileData.frames.length && inputInt >= 0)
+							{setStart(inputInt)}}
+						}/></td> 
 					<td><input className={"timeline-cell"} type={"number"} value={frame} min={"0"} onChange={(e) => 
-						{if (parseInt(e.target.value) < markerFileData.frames.length && parseInt(e.target.value) >= 0) {setFrame(parseInt(e.target.value)); togglePlaying;}}} /></td>
+							{const inputInt = parseInt(e.target.value);
+							if (inputInt < markerFileData.frames.length && inputInt >= 0) 
+							{setFrame(parseInt(e.target.value)); togglePlaying;}}
```

Comment:
is this supposed to be a function call?

Reply:
my bad i accidentally deleted all my changes and had to pull some old code, that was leftover from some testing ill get rid of it